src/block_cache/block_io.c

```diff
@@ -357,6 +479,8 @@ __wt_blkcache_read_multi(WT_SESSION_IMPL *session, WT_ITEM **buf, size_t *buf_co
     if (0) {
 err:
         __wt_free(session, tmp);
+        __wt_scr_free(session, &etmp);
```

Comment:
Do you need to call WT_CLEAR on etmp and ctmp first?

Reply:
No, since `__wt_scr_free` needs to see the `mem` pointer in order to free it.

Reply:
I mean at the start of the function you need to call WT_CLEAR. If they are uninitialized, will we crash here freeing a garbage pointer?

Reply:
Oh right. No, I don't think that can happen - `WT_DECL_ITEM` initialises them to `NULL`.